data/json/npcs/talk_tags.json

```
@@ -376,8 +374,7 @@
       "fella",
       "my dude",
       "buddy",
-      "chum",
-      "bruv"
+      "chum"
```

Brain finally started working, thought up some replacements.
```suggestion
      "chum",
      "wanderer",
      "homie",
      "dawg",
      "bud",
      "pardner"
```
Adds a few alternate spellings of existing lines and a couple new ones.

Adding "compadre" is tempting but its usage in Spanish technically also has the compadre/comadre thing that you just axed amgo for. I'd be tempted to grab a native Spanish speaker to see if anyone actually bothers with the distinction when using it as a term of endearment, I just know English use of it basically always sticks with compadre (whereas we at least sometimes remember the o/a thing since it's more obviously).